### PR TITLE
C109_POJO

### PR DESCRIPTION
**Pull request by vglalitha**

### CODE REVIEW

File: dropwizard-demo/src/main/java/com/dep/Services/TwitterImplement.java [hunk 1]
```diff
@@ -1,56 +1,71 @@
...
 public class TwitterImplement {
-   BRSConfiguration brsConfiguration;
-   ConfigurationBuilder configurationBuilder;
-   TwitterFactory twitterFactory;
-   Twitter twitter;
+    BRSConfiguration brsConfiguration;
+    ConfigurationBuilder configurationBuilder;
+    TwitterFactory twitterFactory;
+    Twitter twitter;
+    TweetRespons tweetRespons;
```
Comment (anasuya1117) [inline dropwizard-demo/src/main/java/com/dep/Services/TwitterImplement.java:22]: This does not needs to be a class variable

File: dropwizard-demo/src/main/java/com/dep/Services/TwitterImplement.java [hunk 2]
```diff
@@ -1,56 +1,71 @@
...
-            List<Status> statuses = twitter.getHomeTimeline();
-            for (int i=0;i<statuses.size();i++){
+
+    public ArrayList<TweetRespons> fetchLatestTweets() {
+        String twitterHandle;
+        String name;
+        String message = null;
+        String profileImageUrl = null;
+        Date createdAt = null;
+
```
Comment (anasuya1117) [inline dropwizard-demo/src/main/java/com/dep/Services/TwitterImplement.java:51]: declaration is not needed, if twitter.getHomeTimeline() does not return anything then these declarations are redundant

File: dropwizard-demo/src/main/java/com/dep/models/TweetRespons.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    public String getCreatedAt() {
+        return createdAt;
+    }
+
+    public String setCreatedAt(String createdAt) {
+        this.createdAt = createdAt;
+        return createdAt;
+    }
```
Comment (anasuya1117) [inline dropwizard-demo/src/main/java/com/dep/models/TweetRespons.java:40]: not used , remove it
same applies for User class

**Review by anasuya1117 — APPROVED** [review]
Please address the comments and merge

File: dropwizard-demo/src/main/java/com/dep/models/TweetRespons.java (hunk 3 above)
Comment (vglalitha) [reply dropwizard-demo/src/main/java/com/dep/models/TweetRespons.java:40]: These are required in the TweetResponse class to fetch the data.

**Pull request merged**